### PR TITLE
Add documentation about widget authentication

### PR DESCRIPTION
**Pull request by andyboeh**
Add documentation about requiring device credentials for widget-initiated service calls

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @andyboeh, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;companion.home-assistant#792).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: docs/integrations/android-widgets.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ This widget will let the user control any media player on their home screen. The
 
 This widget will make the service call when it is tapped. The user will see a green check mark if the call was successful and red if not. Red would indicate either the widget was not setup properly, the Home Assistant server is unavailable or some other error that would be indicated in the Home Assistant log.
 
+You may set the checkbox "Require authentication". If it is set, device credentials (i.e. PIN, pattern, biometrics ...) are required before the service call is initiated.
```
Comment (dshokouhi) [inline docs/integrations/android-widgets.md:53]: Lets add a beta tag at the end of this sentence to signify its a beta feature `<span class="beta">BETA</span>`

**Review by dshokouhi — CHANGES_REQUESTED** [review]
Features are first added to the beta cycle so lets make sure to mention that here, once we cut a new release we will remove the beta tags.

**Review by dshokouhi — APPROVED** [review]
Looks good to me, can be merged once parent is merged

**Pull request merged**